independent-publisher-connectors/HubSpot CMS/settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@troystaylor Please delete `icon.png` and `settings.json` files as these are not required for opensourcing your connector.